hypha/apply/projects/tables.py

```diff
@@ -246,6 +247,51 @@ def order_end_date(self, qs, desc):
         return qs.by_end_date(desc), True
 
 
+class ReportingTable(tables.Table):
+    pk = tables.Column(
+        verbose_name=_("Project #"),
+    )
+    submission_id = tables.Column(
+        verbose_name=_("Submission #"),
+    )
+    title = tables.LinkColumn("funds:projects:detail", args=[tables.utils.A("pk")])
+    organization_name = tables.Column(
+        accessor="submission__organization_name", verbose_name="Organization name"
+    )
+    current_report_status = tables.Column(
+        attrs={"td": {"class": "status"}}, verbose_name="Status"
+    )
+
+    def render_current_report_status(self, value):
+        return format_html("<span>{}</span>", value)
+
+    current_report_submitted_date = tables.Column(
+        verbose_name="Submitted date", accessor="current_report_submitted_date__date"
+    )
+    current_report_due_date = tables.Column(
+        verbose_name="Due Date", accessor="report_config__current_report__end_date"
+    )
+    current_report_last_notified_date = tables.Column(
+        verbose_name="Last Notified",
```

Comment:
Please avoid title case. We have it in several places but are moving away from it.